app/src/scripts/admin/admin.store.js

```diff
@@ -535,6 +597,9 @@ let UserStore = Reflux.createStore({
       let eventLogs = res.body
       this.update({ eventLogs }, () => {
         this.searchLogs('')
+        if (eventLogs.length) {
```

Comment:
A guard like this can just be part of the filterLogs function, it looks like you duplicate it in different areas.